src/main/resources/assets/appliedenergistics2/lang/de_DE.lang

```diff
@@ -25,12 +25,12 @@ tile.appliedenergistics2.quartz_glass.name=Quarzglas
 tile.appliedenergistics2.quartz_vibrant_glass.name=Polarisiertes Quarzglas
 tile.appliedenergistics2.quartz_pillar.name=Certusquarzsäule
 tile.appliedenergistics2.quartz_fixture.name=Geladene Quarzleuchte
-tile.appliedenergistics2.light_detector.name=Quarzlichtsensor
-tile.appliedenergistics2.spatial_io_port.name=IO-Raumport
+tile.appliedenergistics2.light_detector.name=Quarz Lichtsensor
+tile.appliedenergistics2.spatial_io_port.name=IO Port
 tile.appliedenergistics2.spatial_pylon.name=Raumpylon
-tile.appliedenergistics2.tiny_tnt.name=Mini-TNT
+tile.appliedenergistics2.tiny_tnt.name=Mini TNT
 tile.appliedenergistics2.vibration_chamber.name=Vibrationskammer
-tile.appliedenergistics2.wireless_access_point.name=Drahtloser ME-Zugangspunkt
+tile.appliedenergistics2.wireless_access_point.name=ME Drahtloser Zugangspunkt
```

Comment:
The order is somewhat wrong as this is a access point with the additional property of being wireless. Could also be a tethered one.